govcd/org.go

```diff
@@ -49,50 +48,58 @@ func NewAdminOrg(c *Client) *AdminOrg {
 	}
 }
 
-// Given an org with a valid HREF, the function refetches the org
-// and updates the users org data. Otherwise if the function fails,
-// it returns an error
-func (org *Org) Refresh() error {
-	if *org == (Org{}) || org.Org.HREF == "" {
-		return fmt.Errorf("cannot refresh, Object is empty")
+// Generic function that takes in a pointer to a Org or Admin org as an
+// entity and refreshes it by getting the current state from the
+// vcd rest api. Users should use this function after creating,
+// updating, deleting resources under this org. Since the we use
+// org links to get elements within the org, users must refresh
+// after performing an operation that can change the links. Otherwise
+// users might get stale metadata, and for users trying to delete an org
```

Comment:
Thank you for a write-up! Would you add a specific example when this can happen and Refresh should be used?